qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -62,8 +63,15 @@ def run(self, qobj):
         Returns:
             IBMQJob: an instance derived from BaseJob
         """
-        job = IBMQJob(self, None, self._api, qobj=qobj)
+        kwargs = {}
```

Comment:
Non-blocking. I found passing parameters in a dict hard to read, so I'm proposing this to you, although it is a matter of personal taste:
```python
is_new_api = isinstance(self._api, IBMQClient)
job = IBMQJob(
    self, None, self._api, qobj=qobj, use_object_storage=is_new_api, use_websockets=is_new_api)
```

Reply:
Yes, both are sane options - in this case, I went with the kwargs option because I think it signals better "using object storage and/or websockets are a special case: if they don't apply, rely on the default `Job` behavior", which reflects better the present situation.

Reply:
Actually, `rely on the default behaviour` is different of what I was proposing (which is set to `False` explicitely) so, fair enough.